addons/mail/static/src/models/message_view.js

```diff
@@ -565,5 +597,22 @@ registerModel({
             },
             inverse: 'messageViewOwner',
         }),
+        /**
+         * Sender (impersonation note)
+         */
+        senderView: attr({
```

Comment:
(minor: View suffix is typically reserved for parts that instantiate a sub-component, not just a part of the view itself. But it doesn't matter much as everything is gone in master)